### DISCUSSION TITLE
Sperate Volume

### DISCUSSION THREAD

**Discussion by EdgiestNickel89** [tallbl0nde/TriPlayer#63] [Ideas]
Would it be possible to create separate volume controls so games where audio is not important can be muted and just listen to music. Also less important but would it be possible to have a select feature so you can select multiple songs and add to playlist and things like that

**Comment by turohabaneero** [comment]
Most games have options to turn off their audio, no? If the music isn't loud enough you can set every level higher in EQ settings.

**Reply by EdgiestNickel89** [reply]
I forgot about this issue. Anyway, yes a lot do but I was just wondering if there was isolated controls for TriPlayer and everything else
